src/NuGet.Clients/NuGet.PackageManagement.UI/Options/ConfigPathsControl.xaml.cs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+using System.Threading;
+using System.Windows.Controls;
+using System.Windows.Documents;
+using System.Windows.Input;
+using Microsoft.VisualStudio.PlatformUI;
+using NuGet.VisualStudio;
+using NuGet.PackageManagement.Telemetry;
+using NuGet.Common;
+
+namespace NuGet.PackageManagement.UI.Options
+{
+    /// <summary>
+    /// Interaction logic for ConfigPathsControl.xaml
+    /// </summary>
+    public partial class ConfigPathsControl : UserControl
+    {
+        public ConfigPathsWindowViewModel ConfigPathsWindow { get; set; }
```

Comment:
The name `ConfigPathsWindow` doesn't seem correct to me. Any reason why this this property is public?

Reply:
I'll try to come up with a more clear name! `ConfigPathsWindow` is public because it's used by `ConfigPathsControl.xaml`.